hooks/pre/40-el8_modules.rb

```diff
@@ -0,0 +1,7 @@
+if (module_enabled?('katello') || module_enabled?('foreman_proxy_content')) && el8?
+  ensure_dnf_module('pki-core', '10.6')
+
+  if local_postgresql?
+    ensure_dnf_module('postgresql', '12')
```

Comment:
I can take a look at if we can solve this in puppetlabs-postgresql as a long term solution.

Reply:
Could be nice there long term, I dont see modules going away.